pkg/types/intoto/v0.0.1/entry.go

```diff
@@ -92,6 +92,14 @@ func (v V001Entry) IndexKeys() ([]string, error) {
 				result = append(result, alg+":"+ds)
 			}
 		}
+		predicate, err := parseSlsaPredicate(v.env.Payload)
```

Comment:
Perhaps add a comment that not all in-toto statements contain SLSA predicates, and provide some supporting links if there are?

Reply:
done! i figure we can extend this to something like "getPredicateMaterials" in case the SPDX or other predicates also include materials with digest sets.